web/impact/impact/tests/test_algolia_api_key_view.py

```diff
@@ -119,6 +117,19 @@ def test_finalist_user_filter_includes_is_active_filter(self):
             user, self._person_directory_url())
         self.assertIn(IS_ACTIVE_FILTER, response_data["filters"])
 
+    def test_finalist_user_response_has_programs_finalist_in(self):
+        program = ProgramFactory(program_status=ACTIVE_PROGRAM_STATUS)
+        user = self._create_user_with_role_grant(program, UserRole.FINALIST)
+        response_data = self._get_response_data(
+            user, self._mentor_directory_url())
+        self.assertIn(program.name, response_data["finalist_programs"])
+
+    def test_staff_user_response_has_no_programs_finalist_in_field(self):
```

Comment:
I think this test is misleading, does the user response have no `finalist programs` because the user is staff, or because the user isn't a finalist in any program?